src/ethereum/rfc003/erc20_htlc.rs

```diff
@@ -24,12 +24,12 @@ impl Erc20Htlc {
         token_quantity: TokenQuantity,
     ) -> Self {
         let mut contract = CONTRACT_TEMPLATE.to_vec();
-        EthereumTimestamp(expiry).fit_into_placeholder_slice(&mut contract[102..106]);
-        refund_identity.fit_into_placeholder_slice(&mut contract[224..244]);
-        redeem_identity.fit_into_placeholder_slice(&mut contract[157..177]);
         SecretHash(secret_hash).fit_into_placeholder_slice(&mut contract[53..85]);
-        token_contract_address.fit_into_placeholder_slice(&mut contract[307..327]);
-        token_quantity.fit_into_placeholder_slice(&mut contract[261..293]);
+        EthereumTimestamp(expiry).fit_into_placeholder_slice(&mut contract[101..105]);
```

Comment:
:mega: Same here, please don't change the order if not strictly necessary. I believe the order was identical to how it is described in the RFC.

Reply:
Actually I changed it to that order as our `print_offsets` tool prints it that way :)

![image](https://user-images.githubusercontent.com/224613/72583144-49112180-3939-11ea-938a-b865f47d6067.png)

![image](https://user-images.githubusercontent.com/224613/72583157-58906a80-3939-11ea-8119-5aa08d0feb67.png)